lib/routes/tencent/news/author.ts

```diff
@@ -54,44 +54,47 @@ async function handler(ctx): Promise<Data> {
             const author = item.source;
             const abstract = item.abstract;
 
-            return item.articletype === '4'
-                ? {
-                      title,
-                      description: abstract,
-                      link: itemUrl,
-                      author,
-                      pubDate,
-                  }
-                : cache.tryGet(itemUrl, async () => {
-                      const response = await got(itemUrl);
-                      const $ = load(response.data);
-                      const data = JSON.parse(
-                          $('script:contains("window.DATA")')
-                              .text()
-                              .match(/window\.DATA = ({.+});/)[1]
-                      );
-                      const $data = load(data.originContent?.text || '', null, false);
-                      if ($data) {
-                          // Not video page
-                          $data('*')
-                              .contents()
-                              .filter((_, elem) => elem.type === 'comment')
-                              .replaceWith((_, elem) =>
-                                  art(path.join(__dirname, '../templates/news/image.art'), {
-                                      attribute: elem.data.trim(),
-                                      originAttribute: data.originAttribute,
-                                  })
-                              );
-                      }
+            if (item.articletype === '4' || item.articletype === '118') {
+                // Video
+                return {
+                    title,
+                    description: `<a href=${item.url}><img src="${item.articletype === '4' ? item.miniProShareImage : item.miniVideoPic}" style="width: 100%" referrerpolicy="no-referrer"></a>`,
```

Comment:
```suggestion
                    description: `<a href=${item.url}><img src="${item.articletype === '4' ? item.miniProShareImage : item.miniVideoPic}" style="width: 100%"></a>`,
```
This attribute will be added by RSSHub automatically.